examples/api-endpoints/react-email-verification.tsx

```
@@ -0,0 +1,328 @@
+
+  const formatTime = (seconds: number) => {
+    const mins = Math.floor(seconds / 60);
+    const secs = seconds % 60;
+    return `${mins}:${secs.toString().padStart(2, '0')}`;
+  };
+
+  const getTitle = () => {
+    switch (type) {
+      case 'password_reset':
+        return 'Reset Your Password';
+      case 'email_verification':
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)